docs/source/workflows/performance_workflows.rst

```diff
@@ -1,7 +1,7 @@
-Performance workflows
+Performance Workflows
```

Comment:
Should we remove this page entirely since it's deprecated? The getting started page says that there are only 2 workflows.

Reply:
I didn't want to remove the docs until we remove the workflow in case some people are still using performance workflows, but I also didn't want to link new users to it (hence why I left the top-level page saying there were 2 workflows). What if we add a `deprecated` tag to the title similar to what we do for alpha functionality?

Reply:
Deprecated tag sounds good

Reply:
👍